requirements.txt

```diff
@@ -10,3 +10,4 @@ xgboost==0.72.1
 tabulate==0.8.2
 python-igraph==0.7.1.post6
 srcd_smart_open==1.9.0
+git+https://github.com/facebookresearch/fastText
```

Comment:
Really, they don't have a pypi package? Whoa

Reply:
Yep, we did the same in the summer. They have a very restricted Python binding, that's why I would love to get rid of them.